src/pybind/mgr/dashboard/tox.ini

```diff
@@ -47,7 +47,7 @@ commands =
     pytest {posargs}
 
 [testenv:run]
-basepython=python3
+basepython=python3.8
```

Comment:
Shouldn't this be fixed in line 158 instead?
```ini
[testenv:openapi-{check,fix}]
basepython = python3
```
Or even better, fixed for all text environments: by changing that in the `[testenv]` section, which [will be inherited by the remaining](https://tox.wiki/en/latest/config.html#tox-environments) `[testenv:xxx]` sections once the current override `basepython = python3` is removed from each one:
```ini
[testenv]
basepython = py3.8
```
```diff
[testenv:openapi-{check,fix}]
-basepython = python3
```